test-utils/loadtester/src/main.rs

```diff
@@ -29,41 +39,160 @@ fn configure_logging(log_level: log::LevelFilter) {
+                    .default_value("10")
+                    .help("Duration of load test in seconds"))
+            .arg(
+                Arg::with_name("type")
+                    .long("type")
+                    .takes_value(true)
+                    .default_value("set")
+                    .possible_values(&["set", "send_money", "heavy_storage"])
+                    .help("Transaction type")))
+        .subcommand(SubCommand::with_name("create_genesis").about("Create genesis file of many accounts for launch a network")
+            .arg(
+                Arg::with_name("accounts")
+                .long("accounts")
+                .takes_value(true)
+                .default_value("400")
+                .help("Number of accounts to create"))
+            .arg(
+                Arg::with_name("validators")
+                .long("validators")
+                .takes_value(true)
+                .default_value("4")
+                .help("Number of validators to create"))
+            .arg(
+                Arg::with_name("prefix")
+                    .long("prefix")
+                    .takes_value(true)
+                    .default_value("near")
+                    .help("Prefix the account names (account results in {prefix}.0, {prefix}.1, ...)"))
+            .arg(
+                Arg::with_name("home")
+                    .long("home")
+                    .takes_value(true)
+                    .default_value(&default_home)))
+        .get_matches();
+
+    match matches.occurrences_of("v") {
+        0 => configure_logging(log::LevelFilter::Error),
+        1 => configure_logging(log::LevelFilter::Warn),
+        2 => configure_logging(log::LevelFilter::Info),
+        3 | _ => configure_logging(log::LevelFilter::Debug),
+    }
+
+    match matches.subcommand() {
+        ("create_genesis", Some(args)) => create_genesis(args),
+        ("run", Some(args)) => run(args),
+        _ => unreachable!(),
+    }
+}
+
+pub const CONFIG_FILENAME: &str = "config.json";
+
+fn create_genesis(matches: &clap::ArgMatches) {
+    let n = value_t_or_exit!(matches, "accounts", u64) as usize;
+    let v = value_t_or_exit!(matches, "validators", u64) as usize;
+    let prefix = value_t_or_exit!(matches, "prefix", String);
+    let dir_buf = value_t_or_exit!(matches, "home", PathBuf);
+    let dir = dir_buf.as_path();
+
+    let (mut configs, signers, network_signers, genesis_config) =
+        create_testnet_configs(v, n - v, &format!("{}.", prefix), false);
+    for i in 0..v {
+        let node_dir = dir.join(format!("{}.{}", prefix, i));
+        fs::create_dir_all(node_dir.clone()).expect("Failed to create directory");
+
+        signers[i].write_to_file(&node_dir.join(configs[i].validator_key_file.clone()));
+        network_signers[i].write_to_file(&node_dir.join(configs[i].node_key_file.clone()));
+
+        genesis_config.write_to_file(&node_dir.join(configs[i].genesis_file.clone()));
+        configs[i].consensus.min_num_peers = 0;
```

Comment:
Is there a reason for this? It seems reasonable to require at least one peer.

Reply:
With this it can test one node performance (to understand maximum without network maximum). And it doesn't hurt if you want to test more one node. In practice I tested 1, 2, 3, 4, 5, 10 node network, both of them you can wait they have (n-1)/(n-1)/40 peers before running loadtester.

Reply:
you can just use `if v == 1 { 0 } else { 1 }` to special case a single node?

Reply:
Sounds good